shared/src/business/useCases/docketEntry/addPaperFilingInteractor.js

```diff
@@ -107,6 +107,7 @@ exports.addPaperFilingInteractor = async (
       docketNumberWithSuffix: caseToUpdate.docketNumberWithSuffix,
       inProgress: isSavingForLater,
       isRead: user.role !== ROLES.privatePractitioner,
+      leadDocketNumber: caseToUpdate.leadDocketNumber,
```

Comment:
Moving the comment from the old PR to here! Unit test?

Reply:
added test and thank you for the help